ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectAssignment/Index.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@page "/project-assignment/{id:int}"
+@using ApplyToBecomeInternal.Utils
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model ApplyToBecomeInternal.Pages.ProjectAssignment.IndexModel
+@{
+    ViewData["Title"] = "Who will be on this project?";
+}
+
+@section BeforeMain
+    {
+    <a asp-page="@Links.TaskList.Index.Page" asp-route-id="@Model.Id" class="govuk-back-link">Back</a>
+}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <span class="govuk-caption-l">@Model.SchoolName</span>
+            <h1 class="govuk-heading-l">Who will be on this project?</h1>
+
+            <div class="govuk-form-group">
+                <label class="govuk-label" for="delivery-officer">
+                    Delivery officer
+                </label>
+
+                <div id="delivery-officer-assigned-hint" class="govuk-hint">
+                    The delivery officer responsible for this project
+                </div>
+
+                <form method="post">
+                    <input type="hidden" asp-for="@Model.Id" />
+                    <select name="SelectedName" class="autocomplete__menu govuk-select" id="delivery-officer">
+                        <option value="@Model.SelectedDeliveryOfficer">@Model.SelectedDeliveryOfficer</option>
+                        @{
+                        foreach (var deliveryOfficer in Model.DeliveryOfficers)
+                        {
+                            <option value="@deliveryOfficer.FullName">@deliveryOfficer.FullName</option>
+                        }
+                    }
+                </select>
+
+                <div class="govuk-button-group govuk-!-margin-top-5">
+                    <button class="govuk-button" data-module="govuk-button">
+                        Continue
+                    </button>
+                    <input type="hidden" name="UnassignDeliveryOfficer" id="UnassignDeliveryOfficer" />
+                    <span class="govuk-body govuk-!-margin-right-2 govuk-!-margin-left-1">or </span><a href="#" id="unassign-link" class="govuk-link">Unassign the delivery officer</a>
+                </div>
+            </form>
+        </div>
+
+    </div>
+</div>
+
+<script type="application/javascript" asp-add-nonce>
+    var deliveryOfficerSelect = document.querySelector('#delivery-officer');
+    accessibleAutocomplete.enhanceSelectElement({
+        selectElement: deliveryOfficerSelect
```

Comment:
We may want to consider all the options to component gives us for configuring this. 
Examples - https://alphagov.github.io/accessible-autocomplete/examples/

Reply:
Yea, I saw that. Did you have any specific options in mind?

Reply:
The prototype isn't using that option. If I use that and Lisa finds a difference she'll raise it as a bug.

Reply:
Ah fair!